### PR TITLE
feat: build browser bundle

### PR DESCRIPTION
**Pull request by SettingDust**
Depends on: #223

Waiting for https://github.com/WebReflection/linkedom/issues/116
Caused some issues

Edit: fixed, waiting for #223
Edit 1: Done

### CODE REVIEW

**Comment by ndaidong** [comment]
@SettingDust hi friend, please check if this can be merge.

**Comment by SettingDust** [comment]
> @SettingDust hi friend, please check if this can be merge.

Done

File: src/utils/logger.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // utils / logger
 
-import debug from 'debug/src/index.js'
+import debug from 'debug'
```
Comment (SettingDust) [inline src/utils/logger.js:3]: for browser

File: src/utils/extractMetaData.js [hunk 2]
```diff
@@ -55,17 +58,16 @@ export default (html) => {
...
   ]
 
-  const $article = new DOMParser().parseFromString(html, 'text/html')
-
-  entry.title = $article.querySelector('head > title')?.innerText
+  const document = new DOMParser().parseFromString(html, 'text/html')
+  entry.title = document.querySelector('head > title')?.innerText
 
-  $article.getElementsByTagName('link').forEach(node => {
+  Array.from(document.getElementsByTagName('link')).forEach(node => {
```
Comment (SettingDust) [inline src/utils/extractMetaData.js:64]: HTMLCollection

File: test-data/regular-article.html [hunk 3]
```diff
@@ -42,6 +42,7 @@ <h1>Article title here</h1>
           <p class="contentdetail">
             Those cheetahs are nothing more than dogs. A <a href="/dict/watermelon">watermelon</a> is an exuberant kangaroo. An octopus is the tangerine of a grapes? The cherry is a shark. Recent controversy aside, they were lost without the cheerful plum that composed their fox. As far as we can estimate, one cannot separate camels from dynamic hamsters. Those tigers are nothing more than cows! A cow is a squirrel from the right perspective. Their banana was, in this moment, a helpful bear.</p>
           <p>The first fair dog is, in its own way, a lemon.</p>
+          <img src="./orange.png">
```
Comment (SettingDust) [inline test-data/regular-article.html:45]: test for relative image src

File: src/utils/chooseBestUrl.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ import {
   slugify
 } from 'bellajs'
 
-import stringComparison from 'string-comparison'
+import * as stringComparison from 'string-comparison'
```
Comment (SettingDust) [inline src/utils/chooseBestUrl.js:7]: have no default export

**Comment by ndaidong** [comment]
@SettingDust thank you, it's on the way.

**Pull request merged**